lcls2-cc-lib/Library/POUs/FB_L2SI_Flowmeter.TcPOU

```diff
@@ -1,22 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <POU Name="FB_L2SI_Flowmeter" Id="{f4fe6025-68b2-43d4-8493-5a0f33999a1a}" SpecialFunc="None">
-    <Declaration><![CDATA[FUNCTION_BLOCK FB_L2SI_Flowmeter
-VAR
-    {attribute 'pytmc' := '
-        pv: MA
-        io: input
-    '}
-    fRaw AT %I*: INT;
-
-    {attribute 'pytmc' := '
-        pv: FLOW
-        io: input
-    '}
-    fFlowRate: LREAL;
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_L2SI_Flowmeter Extends FB_AnalogInput
+VAR_INPUT
+    fResolution : LREAL := 0.1;
+    fOffset : LREAL;
 END_VAR]]></Declaration>
```

Comment:
This function block has no pytmc pragmas, which I think is the only real functional problem with this PR.

Reply:
@nrwslac do you want to consider this now/soon so we can see the flow readbacks in EPICS or do you prefer you or I handle this at a later date?

Reply:
I guess I didn't attach pytmc pragmas to theses variable because I didn't see a need for them to be modified or viewed in EPICs. I am not against it though, do you think these should be in EPICs?

Reply:
On the other hand modifying the Offset via EPICS would be much easier for fine tuning.

Reply:
This happened here: https://github.com/pcdshub/lcls-twincat-general/pull/82